plugins/modules/ec2_vpc_net_info.py

```diff
@@ -146,6 +146,16 @@
                             description: The CIDR block association state.
                             returned: always
                             type: str
+        owner_id:
+            description: The AWS account which owns the VPC
+            returned: always
+            type: str
+            sample: 123456789012
```

Comment:
```suggestion
            sample: 123456789012
            version_added: 1.4.0
```

Reply:
This has always been there, just not documented